src/extension.ts

```diff
@@ -186,6 +190,16 @@ export async function activate(context: vscode.ExtensionContext) {
 		messageBus,
 	);
 
+	const gitExtension =
+		vscode.extensions.getExtension<GitExtension>('vscode.git');
```

Comment:
@henoktsegayetoptal seems to me you and @rajasegar need to cooperate on both of your PRs so you don't merge conflicting changes.

Reply:
@grzpab i've checked Henok's PR, we dont have conflicting changes, we just access the vscode.git from different places. But it will be probably better to move remote detection logic to the RepositoryService.